controllers/appmesh/cloudmap_controller.go

```diff
@@ -60,10 +66,11 @@ func (r *cloudMapReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 }
 
 func (r *cloudMapReconciler) SetupWithManager(mgr ctrl.Manager) error {
+
 	return ctrl.NewControllerManagedBy(mgr).
 		Named("cloudMap").
 		For(&appmesh.VirtualNode{}).
-		Watches(&source.Kind{Type: &corev1.Pod{}}, r.enqueueRequestsForPodEvents).
+		Watches(&customsource.CustomChannel{Source: r.notificationChannel}, r.enqueueRequestsForPodEvents).
```

Comment:
i think both `customsource` and `custom` package should be replaced with the existing `k8s` package.
so it's like `k8s.NotificationChannel` and `k8s.PodController`